loader_hub/joplin/base.py

```diff
@@ -0,0 +1,105 @@
+"""Joplin reader class.
+
+When Joplin is installed and running it will parse all markdown
+files into a List of Documents.
+
+"""
+from datetime import datetime
+import json
+import os
+from typing import Any, Iterator, List, Optional
+import urllib
+
+from langchain.docstore.document import Document as LCDocument
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+
+LINK_NOTE_TEMPLATE = "joplin://x-callback-url/openNote?id={id}"
+
+
+class JoplinReader(BaseReader):
+    """
+    Reader that fetches notes from Joplin.
+
+    In order to use this reader, you need to have Joplin running with the
+    Web Clipper enabled (look for "Web Clipper" in the app settings).
+
+    To get the access token, you need to go to the Web Clipper options and
```

Comment:
would be nice to put info about JOPLIN_ACCESS_TOKEN in the docstring

also more generally document the args

Reply:
good idea, adding.